app/src/components/integrations/integrationProviders/ldapIntegration/ldapFormFields/inputConditional/inputConditional.scss

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.input-conditional {
+  width: 100%;
+}
+
+.input-wrapper {
+  margin-left: 4px;
+  width: 258px;
+  display: inline-grid;
+}
+
+.dropdown {
+  display: inline-block;
+  width: 98px;
+
+  & > div {
```

Comment:
Once again. Can we not rely on some lib component internals while defining styles?
What is the purpose of this?